src/Command/FixTitleCaseCommand.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+namespace AppBundle\Command;
+
+use AppBundle\Doctrine\DBAL\BatchedConnection;
+use AppBundle\Form\TypeExtension\TextTypeExtension;
+use Doctrine\DBAL\Connection;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class FixTitleCaseCommand extends Command
```

Comment:
Can we have simple functional tests ensuring the command runs without error? and maybe chewk a few outputs.